phinx.json

```diff
@@ -0,0 +1,36 @@
+{
+  "paths": {
+    "migrations": "%%PHINX_CONFIG_DIR%%/migrations"
```

Comment:
I did think about at that but came to the conclusion that I wasn't going to interactively generate the DB config as there is no advantage to it over just editing the provided file. 

The advantage of leaving them in place is they serve as templates for the layout of the various formats for people not using the init command, or with different formats they want to change at a later date. Generating them seemed like overkill for little advantage to me, you may disagree?